.github/workflows/kit-bootstrap.yml

```diff
@@ -0,0 +1,152 @@
+name: This workflow automates the build, testing, and bootstrapping of a test environment on Kubernetes using KIT CLI
+on:
```

Comment:
After examining it, I realized that making it triggered by another workflow can only exercise code lives in the main branch, but can't test against new changes from PR.
It makes more sense to make this test a post-submit test.
It's simple to achieve that, just make the `on` section to below.
```
on:
  push:
    branches: [main]
permissions:
  ...
```

We can add a GH action badge (https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge) in README once we verify everything works fine.